### DISCUSSION TITLE
How to index columns ?

### DISCUSSION THREAD

**Discussion by EricDallAgnol** [atoti/atoti#773] [Q&A] [answered]
Hi team,

In order to optimize performances on a non-key column that is heavily queried, in ActivePivot we can index it as secondary index : https://www.activeviam.com/activepivot/latest/docs/datastore/store_indexing/#indexing-individual-columns 
Is this possible to do that also with Atoti ?

Thanks,
Eric

**Comment by tibdex** [comment, marked as answer]
Hi Eric, the Python API does not allow indexing non-key columns. If this is important for you, can you please open a [support ticket](https://activeviam.atlassian.net)?
